src/Modules/Build.lua

```diff
@@ -156,11 +156,57 @@ function buildMode:Init(dbFileName, buildName, buildXML, convertBuild)
 		end
 	end
 	self.controls.pointDisplay.width = function(control)
-		local used, ascUsed = self.spec:CountAllocNodes()
-		local usedMax = 99 + 22 + (self.calcsTab.mainOutput.ExtraPoints or 0)
-		local ascMax = 8
-		control.str = string.format("%s%3d / %3d   %s%d / %d", used > usedMax and "^1" or "^7", used, usedMax, ascUsed > ascMax and "^1" or "^7", ascUsed, ascMax)
-		control.req = "Required level: "..m_max(1, (100 + used - usedMax))
+		if (self.calcsTab.mainOutput.ExtraPoints == nil) 
+		then 
+		  bandit = 0 
+		else 
+		  bandit = self.calcsTab.mainOutput.ExtraPoints or 0 
+		end
+		
+		local PointsUsed, AscUsed = self.spec:CountAllocNodes()
+		local banditStr = ""
+		local usedMax, ascMax, levelreq = 99 + 22 + bandit, 8, 1
+		
+		-- level per act as of ingame description last waypoint, 10 entries
+		local act = {12,22,32,40,44,50,54,60,64,67}
+		-- skillpoints accumulated without bandits per act, 10 entries
+		local skillpoints = {2,3,5,6,8,11,14,17,19,22}
```

Comment:
I would recommend turning this into a proper table that is keyed by the act number, so down the line you don't have to use a cursor, but use `acts[2].skillPoints` and `acts[2].level`, for example:
```suggestion
		local acts = {
		    [1] = { level = 12, questPoints = 2 },
		}
```